e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -131,6 +132,36 @@ test.describe('Example Imagery Object', () => {
     expect(afterDownPanBoundingBox.y).toBeLessThan(afterUpPanBoundingBox.y);
   });
 
+  test('Can use alt+shift+drag to create a tag', async ({ page }) => {
```

Comment:
No, this is an action item for me or someone else with access. The same test contents should work we just need to use existing telemetry instead of Example Imagery